packages/ui-components/src/sectioned-list/sectioned-list.tsx

```diff
@@ -310,13 +309,13 @@ function SectionedList(props: SectionedListProps): JSX.Element {
             {ReactDOM.createPortal(
                 <ListWrapper
                     {...menuProps}
-                    {...attributes.popper}
+                    {...getFloatingProps()}
+                    ref={mergedRefs}
+                    role="listbox"
```

Comment:
Doesn't `getFloatingProps` include a role?

Reply:
That's true, but that was causing tests to fail. It turns out that it's because downshift already includes those aria labels on the inner component, so we ended up with 2 nested aria comboboxes. 

I moved the reference props to the inner element and supplied a proper `useRole(context, { role: <ROLE> })`. That effectively merges those props which solves this issue, but I'm not sure if having the `ref` and its `{...getReferenceProps()}` attached to different elements is that great either.